packages/core/test/helpers.ts

```diff
@@ -9,7 +9,7 @@ export function dom(html: string): DOMWindow {
   jsdom = new JSDOM(`<!doctype html><html><body>${html}</body></html>`, { runScripts: "dangerously" });
 
   setDocumentResolver(() => jsdom.window.document);
-  removeWrapper = addInteractionWrapper((_, oper) => async () => oper);
+  removeWrapper = addInteractionWrapper(async (_, oper) => oper());
```

Comment:
Is the order of these arguments right? It seems like maybe only certain things need metadata, but _every_ wrapper will need to be able to invoke the chain.